### DISCUSSION TITLE
what happens if I set SMTP_SSL=true and SMTP_EXPLICIT_TLS=true ?

### DISCUSSION THREAD

**Discussion by jonathanmmm** [dani-garcia/vaultwarden#1639] [Q&A] [answered]
I dont know if there can be a MITM attack if I only use STARTTLS, so I tried to use TLS forced.
I checked (enabled) both of these settings in the admin panel:
![image](https://user-images.githubusercontent.com/32403139/116600228-5d5c1080-a929-11eb-863a-df67d21108c6.png)

What happens if I have both ticked? Only with forced it didn't work.

**Comment by BlackDex** [comment, marked as answer]
You need to enable secure smtp also. Else force tls will not be triggered. Maybe we could try and fix that logic a bit.

There should be a note regarding this if you hover the label/text if I'm correct.

**Reply by jonathanmmm** [reply]
Ok, thanks. So everything is fine if I have both set.

The following is written, when I hover over it:
![image](https://user-images.githubusercontent.com/32403139/116606309-ce52f680-a930-11eb-8bfe-3ce3f7d9a334.png)

**Reply by BlackDex** [reply]
Yes, all should be fine.

**Reply by jonathanmmm** [reply]
Either writing something in the note or changing the logic would be good :)
I wrote a pull request for a short description update https://github.com/dani-garcia/vaultwarden/pull/1641.
Changed it also in wiki.
